.github/workflows/actions.yml

```diff
@@ -0,0 +1,65 @@
+name: Run Makefile with Selenium
+
+on:
+  # Schedule to run on the 1st day of each month
```

Comment:
Is it a good idea to run this on the 1st day of each month? I personally would be a bit queasy if some script would have the rights to publish on the main branch without any human doing a sanity check. Especially for this: releases are done only max twice a year, so it's not that much work. 

On the other hand: it will be cool that this project will maintain itself.

Reply:
To clarify: it's not the every month part I'm worries about. It's the "allowing a script to change the main branch without any human supervision" part.

Reply:
@gradedSystem can you make the GH Action open a PR if any change is detected?

Reply:
@cristan yeah i understand this, i mean we can keep it manual as well if you dont like this idea, the thing i was tasked to make sure that this datasets to be fully automated

Reply:
@sabas i think it would be possible